ConfigSpace/api/__init__.py

```diff
@@ -0,0 +1,14 @@
+import ConfigSpace.api.distributions as distributions
+from ConfigSpace.api.distributions import Beta, Distribution, Normal, Uniform
+from ConfigSpace.api.types import Categorical, Float, Int
+
+__all__ = [
+    "distributions",
```

Comment:
Hey, why do you advertise `"distributions"` here and not `"types"`? Or asked differently, is there a reason to advertise `"distributions"` in the first place? I'd be in favor of dropping it here and hiding the internal complexity.

Reply:
The main reason was to allow for users to do something like the following incase `Normal` is already in the namespace
```python

class Normal:
   """User defined class or function"""

from ConfigSpace.api import distributions

Int("a", distribution=distributions.Normal(2, 3))
```

Reply:
But you raise a valid point that if we do that, we should also probably export `types` in a similar fashion.

I would be against dropping it but I'm really not pushed either way.

Reply:
Could you then please export `types`?